src/main/java/com/gmail/woosay333/onlinebookstore/exception/RegistrationException.java

```diff
@@ -0,0 +1,7 @@
+package com.gmail.woosay333.onlinebookstore.exception;
+
+public class RegistrationException extends Exception {
```

Comment:
Why this exception is checked?

Reply:
As i know, checked exceptions should be used for predictable, but unpreventable errors that are reasonable to recover from. I think the new user registration process is just such a case, but I could be wrong. If so, correct me and I'll change the exception type to unchecked.

Reply:
Checked exception should be used when it is requred to make other method know that exception is thrown and we need to catch it and handle, in out case we 'catch' exception in exceptionHandler, and it is redundant to make it checked, it just adds come unnecessary code

Reply:
Understood, thanks